src/components/Answers/Answers.tsx

```diff
@@ -62,19 +75,18 @@ const Answers: React.FC<Iprops> = ({ currentCountry, questions, guess }) => {
       style={
         { backgroundImage: `url(${currentCountry.flag})` }
       }>
-      <div className="gradient-container">
+      <section className="gradient-container">
         <p className="user-guess">
           Your guess was {guess}
         </p>
-        <h3>
-          {answer}
-        </h3>
-        <section className="extra-facts">
+        <h3>{answer}</h3>
+        <article className="extra-facts">
           <p>
             {currentCountry.name} is located in {currentCountry.subregion}.
+            Country has currency of {currency.name} and population speaks {languages}!
```

Comment:
We can reword this whole paragraph to be more conversational. Something like- "{currentCountry.name}, located in {currentCountry.subregion}, speaks {languages} and uses {currency.name} for its currency."